compiler-core/src/language_server/engine.rs

```diff
@@ -688,11 +695,13 @@ fn code_action_fix_names(
                 new_text: correction.to_string(),
             };
 
-            CodeActionBuilder::new(&format!("Rename to {}", correction))
+            let action = CodeActionBuilder::new(&format!("Rename to {}", correction))
                 .kind(lsp_types::CodeActionKind::QUICKFIX)
                 .changes(uri.clone(), vec![edit])
                 .preferred(true)
-                .push_to(actions);
+                .build();
+
+            actions.push(action);
```

Comment:
How come this has changed?

Reply:
Oops my bad I was experimenting with a different API and forgot to undo it 😁